client/web/admin/src/components/Authclient/CAuthclientEditorInfo.vue

```diff
@@ -217,29 +219,32 @@
       >
         <b-form-checkbox
           data-test-id="checkbox-allow-access-to-user-profile"
-          :checked="(resource.scope || []).includes('profile')"
+          :checked="((resource.scope || []).includes('profile'))"
```

Comment:
i dont see a reason for double ((
Same for all other occurrences of this

Reply:
It throws off the highlighting in some code editors mine included. It makes it almost impossible to read. If this does not affect the code in anyway, can we please leave it as is?